mrp_production_estimated_cost/README.rst

```diff
@@ -0,0 +1,96 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================================
+Estimated costs in manufacturing orders
+=======================================
+
+Thanks to this module, you can estimate the cost of your manufacturing orders
+order, split as followed:
+
+* Raw Material cost: an analytic line is generated for each material to
+  be consumed in the order with the product cost.
+* Operators time: one line will be generated for the time recorded by the
```

Comment:
* Operators time: for the time recorded by the operators during each operation, one analytic line will be generated (number of lines to be equal to the number of operators).